flare/gp_from_aimd.py

```diff
@@ -59,70 +59,91 @@
 
 class TrajectoryTrainer:
 
-    def __init__(self, frames: List[Structure],
-                 gp: Union[GaussianProcess, MappedGaussianProcess],
-                 rel_std_tolerance: float = 4,
-                 abs_std_tolerance: float = 1,
-                 abs_force_tolerance: float = 0,
-                 max_force_error: float = inf,
-                 parallel: bool = False,
+    def __init__(self, gp: Union[GaussianProcess, MappedGaussianProcess],
```

Comment:
could you please write something like the backward_attributes and backward_arguments in the GP class to keep track of the change?